source-code/design-system/src/color-system/generateTokens.cts

```diff
@@ -2,7 +2,7 @@ import type { Config } from "./types/config.cjs";
 import type { ColorTokens } from "./types/colorTokens.cjs";
 import { TinyColor } from "@ctrl/tinycolor";
 import type { Color } from "./types/color.cjs";
-
+const surface100Color = 82;
```

Comment:
You should add a comment what this is for. The name of the variable is also incorrect. This is not the surface100 color but the surface 100 "lighten base".

Reply:
Btw, this seems to be an over-abstraction. The benefit of having a dedicated variable is close to null. Only when a value is changed but that happens maybe once (now)? On the flip side, it makes reading the code harder. 

```ts
"surface-200": new TinyColor(neutralColors.neutral[900])
	.lighten(85)
```

is easier to reason about than 

```ts
"surface-200": new TinyColor(neutralColors.neutral[900])
	.lighten(surface100Color - 3)
```

1. Why is something subtracted from a color?
2. What is the resulting value?  

I'll leave it for now. Your call whether you want to remove it or not